package com.pulpy.sharedservices.exceptions;

public final class ExceptionMessages {
    private  ExceptionMessages(){};

    public static final String USER_NOT_FOUND = "User not found or deleted";
    public static final String INVALID_PASSWORD = "Invalid password!";
    public static final String INVALID_OTP = "The given OTP is incorrect";
    public static final String USER_NOT_VERIFIED = "The user is not verified";
    public static final String USER_ALREADY_EXISTS = "User already exists";
    public static final String USER_EXISTS_BUT_NOT_VERIFIED = "The given email id already exists but not verified";
    public static final String NOT_FOUND = "not found";
    public static final String TOKEN_INVALID = "Token not valid";
    public static final String CREATION_FAILED_EXCEPTION = "Creating {0} failed. Please try again";





}
